src/main/java/com/github/swapbook/email/EmailService.java

```diff
@@ -0,0 +1,6 @@
+package com.github.swapbook.email;
+
+public interface EmailService {
+    void sendSimpleMessage(String to, String subject, String text);
```

Comment:
to nie jest jakiś asynchroniczny sposób więc może jak by coś zwracał albo jeśli chcemy żeby to było taki wrzucenie i nich siędzieje co chce to jakiś wyjątek albo coś takiego -> taka szybka myśl raczej do dyskusji a nie do zagłębiania się zbytni -> wersja byle by to napisać nie musi mieć tej poprawki XD

Reply:
no ogólnie nie powinno się zwracać samego statusu wysłania bo tak czy siak byś musiał to obsłużyć wcześniej. Więc void zostaje, ale wyrzuca MessagingException